testsuite/tests/singlecluster/observability/conftest.py

```diff
@@ -0,0 +1,93 @@
+"""Conftest for observability tests"""
+
+import time
+
+import backoff
+import pytest
+from openshift_client import selector
+
+
+@pytest.fixture(scope="module")
+def authorization():
+    """Override the authorization fixture to prevent the creation of an AuthPolicy"""
+    return None
+
+
+@pytest.fixture(scope="module")
+def rate_limit():
+    """Override the rate limit fixture to prevent the creation of an RateLimitPolicy"""
+    return None
+
+
+@pytest.fixture(scope="function")
+def observability_enabled(kuadrant, request):
+    """Enable observability before the test and reset it after"""
+
+    kuadrant.refresh().model.spec.observability = {"enable": True}
+    kuadrant.apply()
+    kuadrant.wait_for_ready()
+    kuadrant.refresh()
```

Comment:
I think you don't need to refresh objects after `wait_for_ready`. It's doing it for you